memory-viz/src/user_functions.ts

```diff
@@ -48,6 +48,9 @@ function draw(
     let objs: DrawnEntity[] | DrawnEntity[][];
 
     if (typeof objects === "string") {
+        if (!fs) {
+            throw new Error("fs module not available in this environment.");
```

Comment:
Let's make this error message more informative: "Could not load file {objects} in this environment."